tests/restv1/PayPal/SingleTicketCest.php

```diff
@@ -405,4 +405,128 @@ public function should_return_401_when_trying_to_access_non_public_ticket( Restv
 		$I->seeResponseCodeIs( 401 );
 		$I->seeResponseIsJson();
 	}
+
+	/**
+	 * It should return 400 when trying to create a ticket with invalid price
+	 *
+	 * @test
+	 */
+	public function should_return_400_when_trying_to_create_a_ticket_ticket_with_invalid_price( Restv1Tester $I ) {
```

Comment:
I like those tests, I would also love to see us test creation with variations of Zero.

Like empty string, "", `0.0` , `0,0`, `000` and ` `

Reply:
More tests were added